### PR TITLE
Dev 2.0.0+changelog

### PR DESCRIPTION
**Pull request by jochenklar**

### CODE REVIEW

**Comment by afuetterer** [comment]
If you touch the changelog: Can you add a "\" here?
https://github.com/rdmorganiser/rdmo/blob/44dcda2f419f6dff269ea5e0128cbcdf0d77006b/CHANGELOG.md?plain=1#L24

GitHub renders this line very strange.
```
* Adjust style for <summary> tag -> * Adjust style for \<summary> tag
```

**Comment by afuetterer** [comment]
Maybe we should add paths to the ci run conditions?
Or you add "[skip ci]" to your commit messages for now?

You dont need to run all the tests when changing the changelog, right?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+  * Add table-like views for all elements, with locking, availability and export buttons
+  * Add nested views for catalogs, sections, pages and question sets
+  * Improve edit views for elements, remove modals
+* Add Site-based permissions for all elements and new editor and reviewer roles
+* Add set_collection to Value model to store if this value was part of a set
+* Add JSON project export and refactor CSV exports
+* Add refresh to option set providers
+* Don't open XML exports in browser, except when EXPORT_CONTENT_DISPOSITION = None
+* Fix new Shibboleth setup
+* Refactor packaging, add pyproject.toml, add pre-commit-config, overhaul testing and CI
```
Comment (afuetterer) [inline CHANGELOG.md:21]: Maybe add "Update Python dependencies"? Earlier changelog entries have such points.

Updated so far are the following:
django
django-compressor
pypandoc

pytest
pytest.cov
pytest-django

**Comment by afuetterer** [comment]
`Adjust style for "<summary>" tag` in GitHub still looks funky:
https://github.com/rdmorganiser/rdmo/blob/74dc1ef537d7f4ef46ab087ab6f9a412358745eb/CHANGELOG.md#rdmo-1100-apr-27-2023

You could use a "\", but of course that is ugly outside of GitHub.

**Comment by afuetterer** [comment]
I just realized that GitHub did not display the backslash in my comment either. So the message was confusing.

I wanted to suggest a `\`.

**Comment by jochenklar** [comment]
💡

**Comment by afuetterer** [comment]
You want to add a note somewhere about "Drop support for EOL Python 3.6 and 3.7"?

**Pull request merged**